provider/docker.go

```diff
@@ -463,6 +463,13 @@ func (provider *Docker) getSticky(container dockerData) string {
 	return "false"
 }
 
+func (provider *Docker) getIsBackendLBSwarm(container dockerData) string {
+	if _, err := getLabel(container, "traefik.backend.loadbalancer.swarm"); err == nil {
+		return "true"
```

Comment:
Yeah, I meant to question that when I noticed that was how the sticky label was handled (getSticky) - I ended up doing the same thing here to be consistent.  Shall I correct them both?